x-pack/plugin/sql/sql-cli/src/main/java/org/elasticsearch/xpack/sql/cli/command/CliSession.java

```diff
@@ -24,6 +24,7 @@
     private String fetchSeparator = "";
     private boolean debug;
     private boolean binary;
+    private boolean lenient = CoreProtocol.FIELD_MULTI_VALUE_LENIENCY;
```

Comment:
There are more and more commands and configuration settings to set. I think it's worth considering taking all these out in a separate class, like `SqlConfiguration` one for example, and the CliSession to contain a reference to this configuration instance.

Reply:
Yes, it makes sense. Just wondering if it makes sense to have a separate PR for this.

Reply:
Done